src/main/scala/mesosphere/marathon/event/http/HttpEventActor.scala

```diff
@@ -1,52 +1,114 @@
 package mesosphere.marathon.event.http
 
+import java.lang.System.currentTimeMillis
```

Comment:
I'd like `Clock` better and I am not a friend of prefix-less imports since they make determining the origin of a method more difficult.